include/splinterdb/splinterdb.h

```diff
@@ -321,4 +321,14 @@ splinterdb_stats_print_lookup(const splinterdb *kvs);
 void
 splinterdb_stats_reset(splinterdb *kvs);
 
+// Set the streams where informational and error messages will be printed.
+//
+// These default to /dev/null and stderr, respectively.
+//
+// To capture info messages on stdout, run:
+//    splinterdb_logs_set_streams(stdout, stderr);
+void
+splinterdb_logs_set_streams(platform_log_handle *info_stream,
```

Comment:
Not a deal breaker, but maybe this goes in `public_platform.h`?

Reply:
Fair.  Should it also get renamed to `platform_set_log_streams()`?